src/base-input.js

```diff
@@ -550,6 +553,14 @@ export default class BaseInput extends LitElement {
     if (this.hasBlurred && this.type === 'credit-card') {
       this.validate();
     }
+
+    // Enables keyup validation
+    const regex = new RegExp("^[a-zA-Z0-9]+$", "u");
+    const key = String.fromCharCode(!evt.charCode ? evt.which : evt.charCode);
+
+    if (this.keyValidate && regex.test(key)) {
+      this.validate();
+    }
```

Comment:
This could potentially call `validate` twice if the input is a credit card. Should this be an `else if` or be combined into the previous `if` statement?